spec/requests/api/requests_spec.rb

```diff
@@ -228,6 +228,22 @@
       expect(response.parsed_body).to include(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes workflow in the request resources" do
+      ems = FactoryGirl.create(:ems_vmware)
+      vm_template = FactoryGirl.create(:template_vmware, :name => "template1", :ext_management_system => ems)
+      request = FactoryGirl.create(:miq_provision_request, :requester => @user, :src_vm_id => vm_template.id, :options => {:owner_email => 'tester@miq.com'})
```

Comment:
I'm wondering if it might be better to use "example.com` as the email domain as "miq.com" is a real but unrelated organization